teams/teams-ps/teams/New-CsTeamsEventsPolicy.md

```diff
@@ -67,6 +67,26 @@ Possible values are:
 
 
 
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowEmailEditing
+This setting governs if a user is allowed to edit the communication emails in Teams Town hall or Webinar Events. 

```

Comment:
```suggestion
This setting governs if a user is allowed to edit the communication emails in Teams Town Hall or webinar events. 

```

Reply:
@JohanFreelancer9 we cannot keep the 'w' as lower case as Webinar itself is an independent product similar to Teams Town hall. So not changing it to lower case